hazelcast/src/main/java/com/hazelcast/spi/impl/securestore/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Contains classes/interfaces for the Hazelcast Secure Store SPI.
+ */
+package com.hazelcast.spi.impl.securestore;
```

Comment:
Is this package needed in OSS side?

Reply:
I had it in the `hot-restart` sub-module first, but after a discussion with @mmedenjak, we decided we wanted to make the concept independent of Hot Restart to allow possible future reuse. Since the APIs must be available to the `hot-restart` sub-module, the only suitable place is in OS.
But having said that, I have no problem with moving the interfaces back under `hot-restart` since they are not used anywhere else at the moment (and can be moved out if really needed).

Reply:
If you already discussed this, I am fine with that. 👍